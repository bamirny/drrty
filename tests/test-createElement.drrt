

def int main(){		/*creating a horizontal line with HTML*/

	makeHeader("This is the first separated header");
	createElement("hr", "","");					
	makeHeader("This is the other separated header");

	return 0;

}



