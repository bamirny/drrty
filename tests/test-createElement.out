<h1>This is the first separated header</h1>
<hr class=""></hr>
<h1>This is the other separated header</h1>
